packages/dynamite/dynamite/lib/src/builder/imports.dart

```diff
@@ -24,7 +25,11 @@ List<Spec> generateImports(final AssetId outputId, final State state) => [
       Directive.import('package:meta/meta.dart'),
       Directive.import('package:universal_io/io.dart'),
       const Code(''),
-      if (state.resolvedTypes.isNotEmpty) ...[
+      if (state.resolvedTypes
+          .where(
+            (final type) => type is TypeResultEnum || (type is TypeResultObject && type.className != 'ContentString'),
```

Comment:
Please move this into a separate method for readability (and maybe reusability as well).

Reply:
What about adding it as a method to the state class?

Reply:
Sounds good